docker/Dockerfile

```diff
@@ -32,7 +32,7 @@ WORKDIR /root
 # Note: due to a bug in microdnf, using the --nodocs option causes an exit code of 141: https://github.com/rpm-software-management/microdnf/issues/50
 RUN mkdir -p /run/user/$UID && \
     microdnf update -y --nodocs 1>/dev/null 2>&1 && \
```

Comment:
Add the parameter `--refresh` to invalidate the internal cache in the package manager.